embedded-files/fuseml-core-deployment.yaml

```diff
@@ -1,4 +1,51 @@
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-core
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-workloads
+rules:
+- apiGroups:
+  - tekton.dev
+  resources:
+  - pipelines
+  - pipelineruns
+  verbs:
+  - create
```

Comment:
I am sorry but after https://github.com/fuseml/fuseml-core/pull/34 merged, FuseML also needs permission to list pipelineruns.

Reply:
fixed